script.js

```diff
@@ -1,103 +1,130 @@
-// Get the modal
+const ghost = document.querySelector('.ghost');
+const pumpkin = document.querySelector('.pumpkin');
+const reaper = document.querySelector('.grim_reaper');
+const lineEight = document.querySelector('.line_eight');
+const start = document.querySelector('.start_area');
+const collision = document.querySelectorAll('.collision');
 const winModal = document.querySelector("#gameWinPopup");
 const loseModal = document.querySelector("#gameLosePopup");
+const winningDoor = document.querySelector('.winningSquare');
 
-//close the modal because there is no restart button
-document.querySelector('#closeLoseModal').addEventListener('click', () => {
-    loseModal.style.display = "none";
-})
+//animation section, shouldnt be in function as consts need to be globally accessible
+const ghostMove = ghost.animate([
+    {transform: 'translateY(0px)'},
+    {transform: 'translateY(150px)'},
+    {transform: 'translateY(0px)'}
+], {
+    duration: 3000,
+    iterations: Infinity
+});
+ghostMove.pause();
+
+const pumpkinMove = pumpkin.animate([
+    {transform: 'translateX(0px)'},
+    {transform: 'translateX(120px)'},
+    {transform: 'translateX(0px)'},
+    {transform: 'translateX(-140px)'},
+    {transform: 'translateX(0px)'}
+], {
+    duration: 6000,
+    iterations: Infinity
+});
+pumpkinMove.pause();
+
+const reaperMove = reaper.animate([
+    {transform: 'translateY(0px)'},
+    {transform: 'translateY(100px)'},
+    {transform: 'translateY(0px)'}
+], {
+    duration: 3000,
+    iterations: Infinity
+});
+reaperMove.pause();
 
-document.querySelector('#closeWinModal').addEventListener('click', () => {
+// code to make splash screen mouse hover
+function homeScreenGhost () {
+    document.querySelector('.start-button').addEventListener('mouseover', (e) => {
+        e.stopPropagation();
+        document.querySelector('.game-logo').animate([
+            {transform: 'translateY(0px)'},
+            {transform: 'translateY(-60px)'},
+            {transform: 'translateY(0px)'}
+        ], {
+            duration: 3000,
+            iterations: Infinity
+        });
+    });
+}
+
+homeScreenGhost();
+
+// code here to activate win modal displaying
+function win() {
+    start.removeEventListener('mouseleave', runGame);
+    collision.forEach((item) => {
+        item.removeEventListener('mouseenter', lose);
+    })
+    winningDoor.removeEventListener('mouseenter', win);
+    winModal.style.display = "block";
+    ghostMove.pause();
+    pumpkinMove.pause();
+    reaperMove.cancel();
+    lineEight.removeEventListener('mouseleave',moveReaper);
+}
+
+// code here to activate lose modal displaying
+function lose() {
```

Comment:
this function appears to be identical to the `win` function apart from which modal is opened. to make it more DRY we could have a `reset` function that does all the common functionality, and then call that function from the `win` and `lose` functions with the line to show the correct modal.